src/test/java/BaseTest.java

```diff
@@ -77,5 +79,5 @@ public static void verifyPlayingSong() {
         WebElement soundBar = driver.findElement(By.cssSelector("div > button:nth-child(2) > div"));
         Assert.assertTrue(soundBar.isDisplayed());
     }
-    }
+   
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/211261927-d0502e99-cfc9-45c3-8fca-85929ec480dd.png)

![image](https://user-images.githubusercontent.com/104761482/211261960-f6af55c1-6455-45ac-98f4-162100bdc292.png)